pubspec.yaml

```diff
@@ -44,4 +44,4 @@ dependency_overrides:
   react:
     git:
       url: https://github.com/cleandart/react-dart.git
-      ref: 5.1.0-wip
+      ref: test-coverage-take-2
```

Comment:
TODO: Remove when https://github.com/cleandart/react-dart/pull/212 is merged into `5.1.0-wip`